### ISSUE TITLE
Unresponsive, API Update

### ISSUE THREAD

**Issue by sbeachx75** [Special-Sharpie/HockeyStatsBot#5]
NHL changed the API base to `https://api-web.nhle.com/v1/` and I think it's resulted in the bot being unresponsive.

**Comment by Special-Sharpie** [comment]
Yes they did, and as a result, none of the original endpoints work anymore. Unfortunately this means that the bot no longer works.
I am hoping to start working a bot v2 that is based off the new API, this however has been slow going as I no longer have the free time I did when the project originally began. 

The idea would be start from the ground and shed much of the old code I wrote as a less experienced developer!

**Comment by Special-Sharpie** [comment]
I would like to follow up, the future of this bot will likely be in a new repo. Once I have a better idea of the future of this project, I will share more information. 

Thanks!
